batch_requests.go

```diff
@@ -135,7 +146,13 @@ func toBatchItem(requestInfo abstractions.RequestInformation) (BatchItem, error)
 		headers[key] = strings.Join(value, ",")
 	}
 	request.SetHeaders(headers)
-	request.SetUrl(&uri.Path)
+
+	baseUri, err := getBaseUrl(br.adapter)
+	if err != nil {
+		return nil, err
+	}
+	var finalUrl = strings.Replace(uri.String(), baseUri.String(), "", 1)
+	request.SetUrl(&finalUrl)
```

Comment:
Gets the url including path parameters and query parameters excluding the base url